### DISCUSSION TITLE
GUI for buying/selling

### DISCUSSION THREAD

**Discussion by daniel-skopek** [PotatoCraft-Studio/QuickShop-Reremake#450] [Ideas]
When I click on the shop sign to buy/sell an item, it asks me to enter the number of items in the chat. If I'm muted, I can't send messages to the chat. It would be good to add that when I click on the shop sign, it opens a GUI where I can select the number of items.

**Comment by sandtechnology** [comment]
/qs amount <input> should solve it

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: daniel-skopek ***@***.***>
Sent: Thursday, February 15, 2024 10:54:30 PM
To: PotatoCraft-Studio/QuickShop-Reremake ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [PotatoCraft-Studio/QuickShop-Reremake] GUI for buying/selling (Discussion #450)


When I click on the shop sign to buy/sell an item, it asks me to enter the number of items in the chat. If I'm muted, I can't send messages to the chat. It would be good to add that when I click on the shop sign, it opens a GUI where I can select the number of items.

**Comment by daniel-skopek** [comment]
Worked, thanks!
